src/GovUk.Education.ExploreEducationStatistics.Common/Services/FileStoragePathUtils.cs

```diff
@@ -67,6 +67,15 @@ public static string PublicContentDataBlockPath(
             return $"{PublicContentDataBlockParentPath(publicationSlug, releaseSlug)}/{dataBlockId}.json";
         }
 
+        public static string PublicContentDataBlockLocationsPath(
+            string publicationSlug,
+            string releaseSlug,
+            Guid dataBlockId,
+            long boundaryLevelId)
+        {
+            return $"{PublicContentDataBlockParentPath(publicationSlug, releaseSlug)}/{dataBlockId}/{boundaryLevelId}-locations.json";
```

Comment:
I think it might be worth asking the rest of their devs on where this should live, because I can imagine people have opinions about it, and is quick and easy to change.

It would be good to have a scheme where anyone looking at the files in blob storage without context can immediately recognise what these cache files are for. So my suggestion would be to change it to `$"{PublicContentDataBlockParentPath(publicationSlug, releaseSlug)}/{dataBlockId}-boundary-levels/{dataBlockId}-{boundaryLevelId}.json";`

This would also be easier to select by regex if we wanted to create/update an endpoint to remove them as suggested in another comment.